sdc/sdc_autogenerated.py

```diff
@@ -308,7 +309,7 @@ def _series_operator_sub_common_impl(self, other):
                     result_data[:] = self._data - other._data
 
                     if none_or_numeric_indexes == True:  # noqa
-                        result_index = left_index.astype(numba_index_common_dtype)
+                        result_index = left_index.astype(numba_n_dtypeleft_index, )
```

Comment:
It's probably a typo here, and moreover np.astype call still remains in this and below lines